apps/src/submitHelper.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * This helper can be used by an app (e.g. GameLab) to handle the pressing of
+ * submit/unsubmit buttons if the level is submittable.
+ */
+
+import commonMsg from '@cdo/locale';
+import dom from './dom';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DialogButtons from './templates/DialogButtons';
+
+// Parameters provided by the calling app.
+let studioApp, onPuzzleComplete, unsubmitUrl;
+
+/**
+ * Set up the handlers for the submit and unsubmit buttons.
+ * The DOM should have submitButton and unsubmitButton by now.
+ * Also store some parameters to be used in these handlers.
+ *
+ * @param {Object} params
+ * @param {Object} params.studioApp - The studioApp itself.
+ * @param {function} params.onPuzzleComplete - Function to call when submitting.
+ * @param {string} params.unsubmitUrl - URL to post to when unsubmitting.
+ */
+export function initializeSubmitHelper(params) {
+  studioApp = params.studioApp;
```

Comment:
Thought we decided we'd prefer to import the singleton instead of passing this in.